rados/ioctx.go

```diff
@@ -118,6 +122,35 @@ func (ioctx *IOContext) SetNamespace(namespace string) {
 	C.rados_ioctx_set_namespace(ioctx.ioctx, c_ns)
 }
 
+// GetNamespace gets the namespace used for objects within this IO context.
+//
+// Implements:
+//  int rados_ioctx_get_namespace(rados_ioctx_t io, char *buf,
+//                                unsigned maxlen);
+func (ioctx *IOContext) GetNamespace() (string, error) {
+	if err := ioctx.validate(); err != nil {
+		return "", err
+	}
+	var (
+		err error
+		buf []byte
+		ret C.int
+	)
+	retry.WithSizes(128, 8192, func(size int) retry.Hint {
+		buf = make([]byte, size)
+		ret = C.rados_ioctx_get_namespace(
```

Comment:
Unfortunately this is not part of Mimic:

    rados/ioctx.go:141:9: could not determine kind of name for C.rados_ioctx_get_namespace

Reply:
Yep. I was halfway through fixing it with build tags yesterday but I never completed it. I will try to get back to it today.